src/redux/books/initialbookstate.js

```diff
@@ -0,0 +1,34 @@
+const initialBookState = [
+  // {
+  //   author: 'Chinua Achebe',
+  //   title: 'Things Fall Apart',
+  //   id: 1,
+  // },
+  // {
+  //   author: 'Hans Christian Andersen',
+  //   title: 'Fairy tales',
+  //   id: 2,
+  // },
+  // {
+  //   author: 'Dante Alighieri',
+  //   title: 'The Divine Comedy',
+  //   id: 3,
+  // },
+  // {
+  //   author: 'Chinua Achebe',
+  //   title: 'Anthills of the savanah',
+  //   id: 4,
+  // },
+  // {
+  //   author: 'Unknown',
+  //   title: 'Around the globe',
+  //   id: 5,
+  // },
+  // {
+  //   author: 'Unknown',
+  //   title: 'One Thousand and One Nights',
+  //   id: 6,
+  // },
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

Reply:
This issue is well resolved! ✔️